### DISCUSSION TITLE
Exploring Medusa Integration with Open Source POS, CRM, and Accounting Systems, etc.

### DISCUSSION THREAD

**Discussion by edardev** [medusajs/medusa#5382] [Feature Requests]
Greetings, Medusa Community! It would greatly enhance our experience if we could explore the possibility of integrating Medusa with Odoo POS or any other open-source POS, CRM, Accounting, and more. Your support and insights are invaluable!

The project at https://www.odoo.com/ appears to be a promising candidate for seamless integration with MedusaJS. Let's explore the potential together!

Best!

**Comment by edardev** [comment]
ERPNext seems to be a genuinely open-source, comprehensive solution that caters to various business needs, including ERP, POS, Payroll, and more.

**Reply by herawais** [reply]
@edardev are you willing to work together and collaborate to release these packages ?

**Reply by edardev** [reply]
Yes

**Reply by Julian-Dumitrascu** [reply]
Willing to talk about [Odoo, Frappe, and Medusa](https://www.ofm.co.za/listenlive).

**Reply by herawais** [reply]
> Willing to talk about [Odoo, Frappe, and Medusa](https://www.ofm.co.za/listenlive).

@Julian-Dumitrascu i would be happy to discuss more

**Reply by EnSeal** [reply]
@edardev were you able to integrate with frappe? I am hoping we can collaborate on this

**Reply by herawais** [reply]
I have integrated with Frappe, 2 way integration, let me know if you need any help.

**Reply by EnSeal** [reply]
Good to hear. I will need plenty of help. We are looking at integrating erpnext with our site to sync orders from the site back into the sync.

If you don't mind, we can agree on your terms for help

**Reply by Julian-Dumitrascu** [reply]
I don't see that GitHub sends every [notification](https://github.com/medusajs/medusa/discussions/5382#discussioncomment-7865644).

**Reply by edardev** [reply]
@herawais I'm excited to know that you've already integrated with Frappe in a two-way integration. I'm very interested in learning more about your implementation and exploring possibilities for collaboration to further extend this integration between Frappe/ERPNext and MudusaJS. Please let me know a convenient time for us to discuss this further.

**Reply by rickylabs** [reply]
@herawais, that's great to hear! Could you share any resources that demonstrate the two-way synchronization process? I'm curious if it incorporates Medusa's new workflow system. I'm really keen to learn about your approach.

In our company, we've deployed N8N to manage the synchronization of products, variants, suppliers, and customers. It's been effective, but as the complexity of tasks increases, we're starting to see the limits of N8N. This has led us to explore integrating Medusa's latest workflow system with N8N.

Our next steps involve extending this synchronization to include orders, stock, and other crucial business elements.

The combination of ERPNext's robustness and Medusa's adaptability is opening exciting opportunities for us. 

With this in mind, I'm looking forward to the possibility of Medusa developing dedicated ERP support, especially plugins that are compatible with ERPNext. Also looking forward to medusa team (@olivermrbl) to give us feedback about an official N8N Medusa node https://github.com/medusajs/medusa/discussions/4804#discussion-5533976

**Reply by Julian-Dumitrascu** [reply]
@edardev @rickylabs We can [talk](https://github.com/Sol-Global-Management/data-management/issues/10) usefully about such topics.

**Reply by edardev** [reply]
Hi @herawais, I need help doing this integration, please let me know if you would like to connect with me to help me complete this integration,  https://cal.com/edardev

**Reply by DeepBhat** [reply]
Hey I wanna help with this integration, what's the status?

**Reply by edardev** [reply]
I'm not entirely sure about the status at the moment. However, @herawais has been active in responding to our comments. I propose an idea: if everyone is on board, we could organize weekly to collectively contribute to this valuable ERP and E-commerce integration using Medudajs and ERPNext. Your thoughts on this? @Julian-Dumitrascu @rickylabs @DeepBhat @EnSeal

**Reply by DeepBhat** [reply]
I think that is a great idea. I am totally on board

**Reply by Julian-Dumitrascu** [reply]
@edardev 1. I might be able to find time for this soon.
2.1 GitHub did not notify me when you mentioned me. It seems they don't notify about mentions made after creating the first version of one's message.
2.2 It is not the first time when their notifications have seemed unreliable, e.g. in this conversation.
3. We can agree on how we organise our interactions.
Examples: 
a. We might use at least one other repository for our conversations.
b. We can use GitHub and other software, depending on what each participant requires.
I am willing to configure e.g. GitHub repositories and ClickUp spaces for us to use, and to host online talks when we want such group talks. We'll see what each participant wants and what plan we make. If somebody else is the host, I'll be a guest, because this is how some software is designed.

**Reply by uc77** [reply]
Guys any progress done here. We're working on something as well so why not all get together?

**Reply by il-dar** [reply]
interested in doing this as well. Also would say Apache ofbiz is a good contender too as it seems the most feature rich with outdated UI on its online store side of things. Seems like they would complement each other very well. Everything else is very robust JAVA framework with flexibility of using Groovy.

**Reply by motiveahmedsaboor** [reply]
@herawais Did integrate the medusa to ERPNext system. Can you share some github repositories so I can implement the same integration in my system as well.

**Comment by Julian-Dumitrascu** [comment]
> discuss

I don't know how I missed the notification about your message.
Thanks to @EnSeal, I'm back in this conversation.
We could [choose some software](https://github.com/Sol-Global-Management/communication/issues/3) to [schedule talks](https://github.com/Sol-Global-Management/data-management/issues/10).

**Reply by Julian-Dumitrascu** [reply]
@herawais I've found no [notification](https://github.com/notifications) about your message.
Did GitHub notify you about my message to which I'm responding?

**Comment by uc77** [comment]
https://discuss.frappe.io/t/request-for-full-frappe-erpnext-integration-with-medusajs/116557/13
There is an ongoing discussion here how we can integrate Medusa JS with the ERPNext POS system.

**Reply by SalahAdDin** [reply]
How is it going on?

**Comment by uc77** [comment]
I'm building a native one that should be part of Medusa V2. Interested to join forces? Shoot us an email at [unifiedcommercehq@gmail.com](mailto:unifiedcommercehq@gmail.com). More details here https://github.com/medusajs/medusa/discussions/9795
